releases/release-4.0.11.md

```diff
@@ -0,0 +1,169 @@
+
++ TiKV
+
+    - Fix the issue that TiKV is failed to build with `PROST=1` [#9604](https://github.com/tikv/tikv/pull/9604)
+    - Fix the unmatched memory diagnostics [#9589](https://github.com/tikv/tikv/pull/9589)
+    - Fix the issue that the end key of a partial RawKV-restore range is inclusive [#9583](https://github.com/tikv/tikv/pull/9583)
+    - Fix the issue that TiKV might panic when loading the old value of a key of a rolled-back transaction during TiCDC's incremental scan [#9569](https://github.com/tikv/tikv/pull/9569)
+    - Fix the configuration glitch of old values when changefeeds with different settings connect to one Region [#9565](https://github.com/tikv/tikv/pull/9565)
+    - Fix a crash issue that occurs when running a TiKV cluster on a machine with a network interface that lacks the MAC address (introduced in v4.0.9) [#9516](https://github.com/tikv/tikv/pull/9516)
+    - Fix the issue of TiKV OOM when backing up a huge Region [#9448](https://github.com/tikv/tikv/pull/9448)
+    - Fix the issue that `region-split-check-diff` cannot be customized [#9530](https://github.com/tikv/tikv/pull/9530)
+    - Fix the issue of TiKV panic when the system time goes back [#9542](https://github.com/tikv/tikv/pull/9542)
+
++ PD
+
+    - Fix the issue that member health metrics are incorrectly displayed [#3368](https://github.com/pingcap/pd/pull/3368)
+    - Forbid removing the tombstone store that still has peers [#3352](https://github.com/pingcap/pd/pull/3352)
+    - Fix the issue that the store limit cannot be persisted [#3403](https://github.com/pingcap/pd/pull/3403)
+    - Fix the limit constriction of the scatter range scheduler [#3401](https://github.com/pingcap/pd/pull/3401)
+
++ TiFlash
+
+    - Fix a bug that the `min`/`max` result is wrong for decimal types
+    - Fix a bug that TiFlash might crash when reading data
+    - Fix the issue that some data written after DDL operations might be lost after data compaction
+    - Fix the issue that TiFlash incorrectly handles decimal constants in Coprocessor
+    - Fix the potential crash during the learner read process
+    - Fix the inconsistent behaviors of division by `0` or `NULL` between TiDB and TiFlash
+
++ Tools
+
+    + TiCDC
+
+        - Fix a bug that the TiCDC service might unexpectedly exit when `ErrTaskStatusNotExists` and the closing of `capture` session occur at the same time [#1240](https://github.com/pingcap/ticdc/pull/1240)
+        - Fix the old value switch of a `changefeed` might be affected by another `changefeed` [#1347](https://github.com/pingcap/ticdc/pull/1347)
+        - Fix a bug that the TiCDC service might hang when processing a new `changefeed` with invalid the `sort-engine` parameter [#1309](https://github.com/pingcap/ticdc/pull/1309)
+        - Fix the issue of panic that occurs when getting the debugging information on none owner nodes [#1349](https://github.com/pingcap/ticdc/pull/1349)
```

Comment:
Please update "none owner nodes" to "non-owner nodes".

Reply:
Done.

Reply:
LGMT